Package.swift

```diff
@@ -14,23 +14,22 @@ let excludeFiles: String = []
 let package = Package(
     name: "Web3swift",
     platforms: [
-        .macOS(.v10_12), .iOS(.v11)
+        .macOS(.v10_12), .iOS(.v15)
```

Comment:
concurrency requires macOS of at least V12

Reply:
It's 10_15 for macOS and 13 for iOS after 13.2 Xcode release that adds back capability.

Reply:
XCode says `12.0.0` for MacOS
![Screen Shot 2022-04-24 at 10 23 09 AM](https://user-images.githubusercontent.com/38660775/164981370-a3bbd85e-c233-475a-825a-32327e9304c6.png)

Reply:
Ignore that, seems I need to update my XCode install.